### PR TITLE
After two weeks in "Review", change status of ERC-681 to "Last Call"

### PR DESCRIPTION
**Pull request by nagydani**
There have been no objections for two weeks while the status was "Review", so it is time to move on to "Last Call".

### CODE REVIEW

File: EIPS/eip-681.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ title: URL Format for Transaction Requests
 author: Daniel A. Nagy (@nagydani)
 type: Standards Track
 category: ERC
-status: Review
+status: Last Call
```
Comment (MicahZoltu) [inline EIPS/eip-681.md:7]:
```suggestion
status: Last Call
review-period-end: 2021-01-29
```

**Review by MicahZoltu — CHANGES_REQUESTED** [review]
```
ethereum_address        = ( "0x" 40*40HEXDIG ) / ENS_NAME
```
I believe this is a typo, should be `40*HEXDIG`?

**Comment by nagydani** [comment]
> ```
> ethereum_address        = ( "0x" 40*40HEXDIG ) / ENS_NAME
> ```
> 
> I believe this is a typo, should be `40*HEXDIG`?

Yes, it is a typo, you are correct.

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - Trying to change EIP 681 state from Review to Last Call

File: EIPS/eip-681.md [hunk 2]
```diff
@@ -4,7 +4,8 @@ title: URL Format for Transaction Requests
...
+review-period-end: 2021-01-30
```
Comment (MicahZoltu) [inline EIPS/eip-681.md:8]:
```suggestion
review-period-end: 2021-02-07
```

**Comment by MicahZoltu** [comment]
I went ahead and adjusted the review period end to be just over 2 weeks from today so we don't have to wait for another round of dialog with the author.  Once CI passes I'll merge as review.

**Pull request merged**